inspirehep/modules/orcid/models.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this licence, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from invenio_db import db
+
+
+class InspireOrcidRecords(db.Model):
+    __tablename__ = 'inspire_orcid_records'
+
+    id = db.Column(db.Integer, primary_key=True, index=True)
+    orcid = db.Column(db.String(160))
+    record_id = db.Column(db.Integer)
```

Comment:
Actually I believe that the primary key in this context is given by the combination `record_id` + `orcid` which is required to retrieve the put codes. With that you then probably don't need the id.

and `record_id` should then be a foreign key to the UUID of the `Record` model. So not an integer (unfortunately...)

Mmh now I wonder what will happen when 2 records are merged :question: